<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBox</class>
 <widget class="QToolBox" name="ToolBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Adobe Arabic</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>医生信息</string>
  </property>
  <property name="currentIndex">
   <number>5</number>
  </property>
  <property name="tabSpacing">
   <number>4</number>
  </property>
  <widget class="QWidget" name="page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>714</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <attribute name="label">
    <string>长期进修医生</string>
   </attribute>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>470</width>
      <height>670</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="batchSize">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>导入</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>714</height>
    </rect>
   </property>
   <attribute name="label">
    <string>短期进修医生</string>
   </attribute>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>470</width>
      <height>670</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="batchSize">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>714</height>
    </rect>
   </property>
   <attribute name="label">
    <string>研究生</string>
   </attribute>
   <widget class="QListWidget" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>470</width>
      <height>670</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="batchSize">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>714</height>
    </rect>
   </property>
   <attribute name="label">
    <string>规培生</string>
   </attribute>
   <widget class="QListWidget" name="listWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>470</width>
      <height>670</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="batchSize">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>714</height>
    </rect>
   </property>
   <attribute name="label">
    <string>住院医</string>
   </attribute>
   <widget class="QListWidget" name="listWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>470</width>
      <height>670</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="batchSize">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>714</height>
    </rect>
   </property>
   <attribute name="label">
    <string>住院医一阶段</string>
   </attribute>
   <widget class="QListWidget" name="listWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>470</width>
      <height>670</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="batchSize">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>ToolBox</receiver>
   <slot>on_pushButton_12_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>clicked()</signal>
   <receiver>ToolBox</receiver>
   <slot>on_pushButton_13_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButton_12_clicked()</slot>
  <slot>on_pushButton_13_clicked()</slot>
 </slots>
</ui>
